projects/elements/src/lib/lazy-elements/lazy-elements.module.ts

```diff
@@ -1,11 +1,51 @@
-import { NgModule } from '@angular/core';
+import { NgModule, Optional, Inject, ModuleWithProviders } from '@angular/core';
 import { CommonModule } from '@angular/common';
 
 import { LazyElementDirective } from './lazy-element/lazy-element.directive';
+import { ELEMENTS_CONFIG } from './lazy-elements.injectors';
+import {
+  LazyElementsLoaderService,
+  ElementConfig
+} from './lazy-elements-loader.service';
 
 @NgModule({
   declarations: [LazyElementDirective],
   imports: [CommonModule],
   exports: [LazyElementDirective]
 })
-export class LazyElementsModule {}
+export class LazyElementsModule {
+  static forRoot(config: ElementConfig[]): ModuleWithProviders {
+    return {
+      ngModule: LazyElementsModule,
+      providers: [
+        {
+          provide: ELEMENTS_CONFIG,
+          useValue: config ? config : [],
+          multi: true
```

Comment:
So the `ELEMENTS_CONFIG` gives us array of `ElementConfig[]` but is also `multi: true` at the same time...

So that means that  `@Inject(ELEMENTS_CONFIG)` gives us array of `ElementConfig[]` arrays so in other words `ElementConfig[][]` right ?